bin/juttle-engine

```diff
@@ -7,7 +7,7 @@ var engine = require('..');
 
 var defaults = {
     'port': 8080,
-    'root': '/',
+    'root': process.cwd(),
```

Comment:
Not sure what behavior we want exactly, but this change prevents you from being able to run juttles outside of the `examples` dir.